#request type
uint8 REGISTER   = 0
uint8 UNREGISTER = 1

#dependency constants
uint8 SERVICE = 2
uint8 ACTION  = 3
uint8 TOPIC   = 4

#Direction for: Service/Action
uint8 SERVER = 5
uint8 CLIENT = 6

#Direction for: Topic
uint8 PUBLISH   = 7
uint8 SUBS      = 8

#To unregister all dependencies of the respective
#node
uint8 ALL       = 9

#request fields
uint8 requestType
uint8 direction
uint8 dependency
string nodeName
string depName

---

uint8 OK = 0
uint8 ERROR = 1
uint8 result
